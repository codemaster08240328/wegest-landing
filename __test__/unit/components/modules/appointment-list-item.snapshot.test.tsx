import renderer from 'react-test-renderer'
import AppointmentListItem from '@/components/modules/appointment-list-item'

describe('<AppointmentListItem> snapshot', () => {
  it('renders properly with all required props', () => {
    expect(
      renderer
        .create(
          <AppointmentListItem
            start='08:25'
            end='08:50'
            customer='Tony Stark'
            service='Hair Cut'
            color='darkgreen'
            onClick={() => null}
          />,
        )
        .toJSON(),
    ).toMatchSnapshot()
  })

  it('renders with a class', () => {
    expect(
      renderer
        .create(
          <AppointmentListItem
            className='class-test'
            start='08:25'
            end='08:50'
            customer='Tony Stark'
            service='Hair Cut'
            color='darkgreen'
            onClick={() => null}
          />,
        )
        .toJSON(),
    ).toMatchSnapshot()
  })
})
