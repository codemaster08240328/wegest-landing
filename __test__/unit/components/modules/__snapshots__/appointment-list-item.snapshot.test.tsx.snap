// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppointmentListItem> snapshot renders properly with all required props 1`] = `
<div
  className="makeStyles-root-1 "
>
  <div
    className="makeStyles-timeSlot-2"
    data-testid="timeSlot"
    style={
      Object {
        "background": "darkgreen",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      08:25
    </p>
    <span
      className="MuiTypography-root MuiTypography-body1"
    >
      08:50
    </span>
  </div>
  <div
    className="makeStyles-appointment-3"
    data-testid="appointment"
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      Tony Stark
    </p>
    <span
      className="MuiTypography-root MuiTypography-body1"
    >
      Hair Cut
    </span>
  </div>
  <div
    className="makeStyles-action-4"
  >
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-5 Component-root-6"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root material-icons-outlined"
          data-testid="el-icon"
        >
          info
        </span>
      </span>
    </button>
  </div>
</div>
`;

exports[`<AppointmentListItem> snapshot renders with a class 1`] = `
<div
  className="makeStyles-root-1 class-test"
>
  <div
    className="makeStyles-timeSlot-2"
    data-testid="timeSlot"
    style={
      Object {
        "background": "darkgreen",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      08:25
    </p>
    <span
      className="MuiTypography-root MuiTypography-body1"
    >
      08:50
    </span>
  </div>
  <div
    className="makeStyles-appointment-3"
    data-testid="appointment"
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      Tony Stark
    </p>
    <span
      className="MuiTypography-root MuiTypography-body1"
    >
      Hair Cut
    </span>
  </div>
  <div
    className="makeStyles-action-4"
  >
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-5 Component-root-7"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root material-icons-outlined"
          data-testid="el-icon"
        >
          info
        </span>
      </span>
      <span
        className="MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
